spec.html

```diff
@@ -5918,7 +5918,7 @@ <h1>
             1. Else,
               1. Assert: _nx_ is a BigInt.
               1. Return BigInt::lessThan(_nx_, _ny_).
-          1. Assert: _nx_ is a BigInt and _ny_ is a Number, or _nx_ is a Number and _ny_ is a BigInt.
+          1. Assert: _nx_ is a BigInt and _ny_ is a Number, or if _nx_ is a Number and _ny_ is a BigInt.
```

Comment:
Can you drop the two asserts from this commit? This doesn't read right.

Reply:
Yup, done.

One alternative would be `Assert: A and B, or else C and D.`